pinot-common/src/main/java/org/apache/pinot/common/metrics/BrokerMeter.java

```diff
@@ -51,6 +53,7 @@ public enum BrokerMeter implements AbstractMetrics.Meter {
   DATA_TABLE_DESERIALIZATION_EXCEPTIONS("exceptions", false),
   // Reduce responses phase.
   RESPONSE_MERGE_EXCEPTIONS("exceptions", false),
+  HEAP_CRITICAL_LEVEL_EXCEEDED("count", true),
```

Comment:
Don't need to track when other levels exceeded ?

Reply:
Alarming level exceed is actually okay so added panic level metric.